### PR TITLE
doc: optimize the comment of the Request.Done method

### PR DESCRIPTION
**Pull request by li-jin-gou**
PR for https://github.com/valyala/fasthttp/issues/1219

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -2697,6 +2697,9 @@ func (ctx *RequestCtx) Deadline() (deadline time.Time, ok bool) {
 // Done returns a channel that's closed when work done on behalf of this
 // context should be canceled. Done may return nil if this context can
 // never be canceled. Successive calls to Done return the same value.
+//
+// Note: Because creating a new channel for every request is just too expensive so
```
Comment (erikdubbelboer) [inline server.go:2701]:
```suggestion
// Note: Because creating a new channel for every request is just too expensive,
```

File: server.go [hunk 2]
```diff
@@ -2707,6 +2710,9 @@ func (ctx *RequestCtx) Done() <-chan struct{} {
 // If Done is closed, Err returns a non-nil error explaining why:
 // Canceled if the context was canceled (via server Shutdown)
 // or DeadlineExceeded if the context's deadline passed.
+//
+// Note: Because creating a new channel for every request is just too expensive so
```
Comment (erikdubbelboer) [inline server.go:2714]:
```suggestion
// Note: Because creating a new channel for every request is just too expensive,
```

**Review by erikdubbelboer — CHANGES_REQUESTED** [review]
Good comment, but just one small change.

**Comment by li-jin-gou** [comment]
> Good comment, but just one small change.

done

**Comment by erikdubbelboer** [comment]
Thanks!

**Pull request merged**